### PR TITLE
[testbed] increase timeout values in large file tests

### PR DESCRIPTION
**Pull request by VihasMakwana**
See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35053#issuecomment-2334420947

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35053

### CODE REVIEW

**Comment by VihasMakwana** [comment]
Let's wait for https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35055

**Review by crobert-1 — APPROVED** [review]

**Comment by VihasMakwana** [comment]
@open-telemetry/collector-contrib-approvers can we merge this? This would unblock main CI failures for load tests

**Pull request merged**
